commons_api/templates/base.html

```diff
@@ -26,6 +26,7 @@
             <a class="pure-menu-heading" href="/">Democratic Commons <em>demo</em></a>
 <ul class="pure-menu-list">
         <li class="pure-menu-item"><a href="{% url "wikidata:country-list" %}" class="pure-menu-link">Countries</a></li>
+        <li class="pure-menu-item"><a href="{% url "queue-status" %}" class="pure-menu-link">Queue status</a></li>
```

Comment:
This seems a bit more prominent than I'd expect - perhaps a footer link, so it's available openly for those as care but isn't "showcased"?

Reply:
I agree. Shall I put it alongside the language select?